packages/core/src/common/i18n/i18n.spec.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import fs = require('fs');
+import "mocha";
+import * as chai from "chai";
+const Globalize = require("globalize");
+
+const expect = chai.expect;
+
+beforeEach(() => {
+});
+
+// describe('internationalization', () => {
+
+describe('01 Some menu labels', () => {
```

Comment:
In the other test, It looks like they were a number in front of it to identify the test, It is not needed for the Globalize test, remove the numbering for all tests added with this request.